### PR TITLE
Modal added

### PR DESCRIPTION
**Pull request by muskan-gupta-0803**
modal component and docs added

### CODE REVIEW

File: components/Modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+@import "/docs.css";
+@import "../Utilities/utilities.css";
+@import "../Button/button.css";
+
+.modal{
+    position: absolute;
+    top:20%;
+    left:50%;
+    margin-left:-15vw;
+    width:30vw;
```
Comment (HrishikeshMahalle) [inline components/Modal/modal.css:10]: Avoid using hard coded Values

File: components/Modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+.modal-footer{
+    display: flex;
+    justify-content: end;
+}
+
+.close-icon{
+    font-size: 1.5rem;
+    color: grey;
```
Comment (HrishikeshMahalle) [inline components/Modal/modal.css]: Try using Hex color extension.. since its text color variable would be a great choice

**Review by HrishikeshMahalle — APPROVED** [review]
Every thing looks fine. Have a look at minor changes. Good Job!!

File: components/Modal/modal.css (hunk 1 above)
Comment (muskan-gupta-0803) [reply components/Modal/modal.css:10]: okay

**Comment by muskan-gupta-0803** [comment]
> Every thing looks fine. Have a look at minor changes. Good Job!!

thanks for the review.

File: components/Modal/modal.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+var closeIcon=document.querySelector(".close-icon");
```
Comment (ShoaibTamboli) [inline components/Modal/modal.js:1]: avoid var, instead use const, let

**Review by ShoaibTamboli — COMMENTED** [review]
looks good to me!

File: components/Modal/modal.js (hunk 3 above)
Comment (muskan-gupta-0803) [reply components/Modal/modal.js:1]: okay, will change

**Pull request merged**